bluepyefe/nwbreader.py

```diff
@@ -277,6 +277,16 @@ def read(self):
             data (list of dict): list of traces
         """
         data = []
+
+        # Only return data if target_protocols is None or includes "step"
+        if self.target_protocols:
+            allowed = [p.lower() for p in self.target_protocols]
+            if not any(proto in allowed for proto in ["step"]):
```

Comment:
The user will have to provide the protocol "step" in the target protocols explicitly for this reader to work which is fine.